pom-client.xml

```diff
@@ -103,18 +103,35 @@
 				<version>3.1.11</version>
 			</plugin>
 
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-checkstyle-plugin</artifactId>
+				<version>3.1.2</version>
+				<configuration>
+					<configLocation>sbking_checkstyle.xml</configLocation>
+					<encoding>UTF-8</encoding>
+					<failOnViolation>true</failOnViolation>
+					<enableFilesSummary>false</enableFilesSummary>
+					<logViolationsToConsole>true</logViolationsToConsole>
+				</configuration>
+				<!-- uncomment to make build fail on error -->
+				<!-- <executions>
+					<execution>
+						<id>checkstyle</id>
+						<phase>validate</phase>
+						<goals>
+							<goal>check</goal>
+						</goals>
+					</execution>
+				</executions> -->
+			</plugin>
+
 		</plugins>
 	</build>
 
 	<reporting>
 		<plugins>
-			<!-- Bug Finding with SpotBugs -->
```

Comment:
Are you sure about this spotbugs change?

Reply:
As per commented on the PR, I reinstated both spotbugs configuration on the report tags